model/messagedeletions.go

```diff
@@ -0,0 +1,31 @@
+package model
+
+// MessageDeletions holds information about deleted messages.
+type MessageDeletions struct {
+	Messages []*Message
+}
+
+// ToExternal converts the event into an external representation.
+func (del MessageDeletions) ToExternal() interface{} {
+	res := &MessageDeletionsExternal{
+		make([]uint, len(del.Messages)),
+	}
+	for key, value := range del.Messages {
+		res.IDs[key] = value.ID
+	}
+	return res
+}
+
+// MessageDeletionsExternal Model
+//
+// MessageDeletionsExternal holds information about deletions which will be sent to the clients.
```

Comment:
I think we can omit will be "set to the clients". The description should only represent the event not who receives it.

Reply:
Does the same apply to the documentation for `MessageExternal` then? I was trying to emphasize the difference that the `External` suffix makes.

Reply:
I'd say yes, I prefer that the documentation contains information about what the struct represents and not how its used. But this is only a nice to have :D.